### ISSUE TITLE
Add links to source repositories

### ISSUE THREAD

**Issue by JPrevost** [MITLibraries/oastats-ui#136]
Since both repositories we use for this project are open source, it might be useful to provide links to them from the UI... maybe in the footer?

https://github.com/MITLibraries/oastats-ui

https://github.com/MITLibraries/oastats-backend

**Comment by matt-bernhardt** [comment]
Good idea. I'll make this happen in the next few days. Want to review the PR?

**Comment by JPrevost** [comment]
Sure.
